Formula/riff.rb

```diff
@@ -1,10 +1,10 @@
 class Riff < Formula
   desc "A tool that automatically provides external dependencies for software projects"
   homepage "https://github.com/DeterminateSystems/riff"
-  version "1.0.0"
+  version "1.0.1"
   head "https://github.com/DeterminateSystems/riff.git", branch: "main"
-  url "https://github.com/DeterminateSystems/riff/releases/download/v1.0.0/riff-x86_64-darwin"
-  sha256 "91f0ec51fc99f4a34240d54def73b2066d973fee48d8579bc775be7d63284bd1"
+  url "https://github.com/DeterminateSystems/riff/releases/download/v#{version}/riff-x86_64-darwin"
```

Comment:
There _is_ a mechanism for this called "bottles" but I'm not sure if they're available in "tap" formulae, i.e. non-official ones. But I'll look into it because that would of course be a step up.